### PR TITLE
Productionise and test chirps

### PR DESCRIPTION
**Pull request by alexgleith**

### CODE REVIEW

File: data/tools/chirps.py [hunk 1]
```diff
@@ -20,61 +25,81 @@ def download_and_cog_chirps(
...
         exit(1)
 
 
-@click.command("download-chirps-rainfall")
+@click.command("download-chirps")
+@click.option("--year", default="2020")
+@click.option("--month", default="01")
 @click.option(
-    "--s3_dst", default="s3://deafrica-input-datasets/chirps_rainfall_monthly/"
+    "--s3_dst", default="s3://deafrica-data-dev-af/chirps_rainfall/"
```
Comment (cbur24) [inline data/tools/chirps.py]: Is this pathway robust if we decide in the future to index `chirps_daily` data?

File: monitoring/tools/utils.py [hunk 2]
```diff
@@ -6,11 +6,12 @@
 from odc.aws import s3_client, s3_fetch, s3_ls_dir
 
 
-def setup_logging() -> logging.Logger:
+def setup_logging(level: int = logging.INFO) -> logging.Logger:
```
Comment (carioca-au) [inline monitoring/tools/utils.py:9]: If the utils will be shared between **monitoring** and **data**, shouldn't we take it out of **monitoring**?

**Review by carioca-au — APPROVED** [review]
I reckon we could organize better utils.

**Review by carioca-au — CHANGES_REQUESTED** [review]
Apply black, formatting is falling.

File: data/tools/chirps.py (hunk 1 above)
Comment (alexgleith) [reply data/tools/chirps.py]: Hmmm. Ok, I'll change it.

File: monitoring/tools/utils.py (hunk 2 above)
Comment (alexgleith) [reply monitoring/tools/utils.py:9]: Yes we should. I don't like how we have structured it.

**Pull request merged**